### PR TITLE
Maya: Bug fix the fbx animation export errored out when the skeletonAnim set is empty

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
Resolve this bug [discord](https://discord.com/channels/517362899170230292/563751989075378201/1171345794121609226)
If the skeletonAnim SET is empty and fbx animation collect, the fbx animation extractor would skip the fbx extraction
## Additional info
n/a

## Testing notes:
1. Load your rigs
2. It will create the animation family automatically
3. Enable Collect Fbx Animation
4. Publish

### CODE REVIEW

**Review by LiborBatek — APPROVED** [review]
Works like charm! Note the skipping info for collect animation - FBX
image

**Comment by BigRoy** [comment]
> Works like charm! Note the skipping info for collect animation - FBX image

These should **not** be INFO logs! :)

File: openpype/hosts/maya/plugins/publish/extract_fbx_animation.py [hunk 1]
```diff
@@ -42,6 +42,12 @@ def process(self, instance):
         # Export from the rig's namespace so that the exported
         # FBX does not include the namespace but preserves the node
         # names as existing in the rig workfile
+        if not out_members:
+            self.log.debug(
+                "Top group of animated skeleton not found.."
+                "skipping extraction")
+            return
```
Comment (BigRoy) [inline openpype/hosts/maya/plugins/publish/extract_fbx_animation.py:49]:
```suggestion
        if not out_members:
            self.log.debug(
                "Top group of animated skeleton not found. "
                "Skipping FBX animation extraction.")
            return
```

Comment (moonyuet) [reply openpype/hosts/maya/plugins/publish/extract_fbx_animation.py:49]: This extractor is only active when users enable the fbx animation collector. Maybe we should improve the debug message a little bit to allow users understand why the fbx export is skipping.(e.g.Top group of animation skeleton is not found in {set}, Skipping Fbx animation extraction)

File: openpype/hosts/maya/plugins/publish/extract_fbx_animation.py [hunk 2]
```diff
@@ -42,6 +42,16 @@ def process(self, instance):
...
         # FBX does not include the namespace but preserves the node
         # names as existing in the rig workfile
+        if not out_members:
+            skeleton_set = [
+                i for i in instance
+                if i.endswith("skeletonAnim_SET")
+            ]
+            self.log.debug(
+                "Top group of animated skeleton not found in "
+                "{}.\nSkipping fbx animation extraction".format(skeleton_set))
```
Comment (BigRoy) [inline openpype/hosts/maya/plugins/publish/extract_fbx_animation.py:52]:
```suggestion
                "{}.\nSkipping fbx animation extraction.".format(skeleton_set))
```

**Review by BigRoy — APPROVED** [review]
Looks fine with me - didn't test it.

**Comment by moonyuet** [comment]
@LiborBatek can you test again to ensure there is no issue before the merge? Thanks!

**Review by LiborBatek — APPROVED** [review]
All working without any issues...
image

**Pull request merged**